MVCFinal/src/main/java/Controlador/ControladoresPaneles.java

```diff
@@ -0,0 +1,40 @@
+package Controlador;
+
+import Modelo.Modelo;
+import Vista.Vista;
+
+public abstract class ControladoresPaneles {
+
+	private Modelo modelo;
+	private Vista vista;
+	private Controlador controlador;
+	
+	public ControladoresPaneles(Modelo modelo, Vista vista, Controlador controlador) {
+		this.modelo = modelo;
+		this.vista = vista;
+		this.controlador = controlador;
+	}
+
+	public Modelo getModelo() {
+		return modelo;
+	}
+
+	public Vista getVista() {
+		return vista;
+	}
+
+	public Controlador getControlador() {
+		return controlador;
+	}
+
+	public void accionadoBottonVolverPanelPrincipal() {
+		//Utilizamos getters para acceder a los propios Modelo y controlador de la clase
+		//ya que a la hora de testear no podemos instanciar esta clase ya que es abstracta
```

Comment:
En el caso de tener una clase abstracta, como excepción mockeamos la propia clase a testear. Ya que, no tenemos otra manera de resolverlo.

Reply:
Esto mismo hemos hecho, para testear ese método necesitamos crear mocks para algunos metodos que no son de la propia clase. Al ser abstracta no podemos acceder al modelo si no es mediante un getter.

Reply:
Yo diría que esto lo podemos apañar con un spy un make como hacemos en los controladores. Lo habéis probado?